### PR TITLE
chore(infra): Tune pod resources and lower expenses

### PR DESCRIPTION
**Pull request by peturgq**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by brynjarorng — APPROVED** [review]
LGTM

File: apps/application-system/form/infra/application-system-form.ts [hunk 1]
```diff
@@ -27,6 +27,16 @@ export const serviceSetup = (services: {
       SI_PUBLIC_DD_RUM_APPLICATION_ID: '/k8s/DD_RUM_APPLICATION_ID',
       SI_PUBLIC_DD_RUM_CLIENT_TOKEN: '/k8s/DD_RUM_CLIENT_TOKEN',
     })
+    .resources({
+      limits: {
+        cpu: '200m',
+        memory: '256Mi',
```
Comment (AndesKrrrrrrrrrrr) [inline apps/application-system/form/infra/application-system-form.ts:33]: Is this enough? Seems pretty low for a limit

Comment (peturgq) [reply apps/application-system/form/infra/application-system-form.ts:33]: What are you basing this on?

**Comment by peturgq** [comment]
> Lint plz
What is unlinted?

**Pull request merged**